src/components/vehicleReference/index.tsx

```diff
@@ -27,19 +28,30 @@ const VehicleReference: FC<Props> = ({
   const styles = useMultiStyleConfig(`VehicleReference`);
 
   return (
-    <Stack direction={{ xs: 'row', lg: 'column' }} spacing="md">
-      <AspectRatio minW="2xl" ratio={4 / 3} borderRadius="sm" overflow="hidden">
-        {image}
-      </AspectRatio>
-      <Stack spacing={{ xs: 'xs', lg: 'md' }} justify="center">
-        <chakra.h1 __css={styles.carTitle}>{vehicleTitle}</chakra.h1>
-        <chakra.span __css={styles.price}>{price}</chakra.span>
-        <Box>
-          <chakra.p __css={styles.dealerName}>{sellerName}</chakra.p>
-          <chakra.p __css={styles.dealerAddress}>{sellerAddress}</chakra.p>
-        </Box>
+    <article>
+      <Stack direction={{ xs: 'row', lg: 'column' }} spacing="md">
+        <AspectRatio
+          minW="2xl"
+          ratio={4 / 3}
+          borderRadius="sm"
+          overflow="hidden"
+        >
+          {image ? (
+            image
+          ) : (
+            <Image fallbackSrc="assets/images/MissingImage.png" />
```

Comment:
Does this work in the web application?

Reply:
It doesn't work as png in the web application, but I can convert it to svg.

Reply:
I have checked in the next example and now the missing images is displayed. Here is an example.
Thanks! 🌮

<img width="905" alt="Screenshot 2022-07-29 at 19 43 02" src="https://user-images.githubusercontent.com/37380787/181815840-7a36425a-4eb8-4ef9-a45f-16edccdf3105.png">